README.md

```diff
@@ -58,6 +58,19 @@ ReactDOM.render(
 );
 ```
 
+Defer starting of the client: 
```

Comment:
```suggestion
To defer starting the client:
```

When you defer it, how do you start the client later? Rerender with the `startClient` prop set to `true` or is there another way?